### PR TITLE
Adding extra handling for 'type' missing in 'thermostat_json'.

### PR DESCRIPTION
**Pull request by jeroenterheerdt**
I have been using this package as part of my Home Assistant integration but it was broken. I was able to trace it back to not enough checks if 'type' existed in the thermostat_json. Adding those extra checks in this PR. Have not touched anything else.

### CODE REVIEW

**Comment by jeroenterheerdt** [comment]
hope you can merge this and update the python package so the Home Assistant integration can work again :)

**Comment by ryannazaretian** [comment]
I don't see any issues with the primary change. Please remove your configuration file from the commit.

**Comment by jeroenterheerdt** [comment]
oops... removed the conf file now.

File: nexia/home.py [hunk 1]
```diff
@@ -247,10 +247,10 @@ def _update_devices(self):
         if self.thermostats is None:
             self.thermostats = []
             for thermostat_json in self.devices_json:
-                if "thermostat" not in thermostat_json["type"]:
+                t_type = thermostat_json.get("type")
+                if t_type and "thermostat" not in t_type:
```
Comment (bdraco) [inline nexia/home.py:251]:
```suggestion
                if "type" in thermostat_json and "thermostat" not in thermostat_json["type"]:
```

File: nexia/home.py [hunk 2]
```diff
@@ -260,7 +260,8 @@ def _update_devices(self):
 
         thermostat_updates_by_id = {}
         for thermostat_json in self.devices_json:
-            if "thermostat" not in thermostat_json["type"]:
+            t_type = thermostat_json.get("type")
+            if t_type and "thermostat" not in t_type:
```
Comment (bdraco) [inline nexia/home.py:264]:
```suggestion
            if "type" in thermostat_json and "thermostat" not in thermostat_json["type"]:
```

**Review by bdraco — APPROVED** [review]

**Pull request merged**